openedx/adg/lms/helpers.py

```diff
@@ -0,0 +1,16 @@
+"""
+ADG LMS Helpers
+"""
+
+
```

Comment:
I think you have missed to remove these empty lines.

Reply:
Removed